src/NuGet.Clients/NuGet.MSSigning.Extensions/RepoSignCommand.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.CommandLine;
+using NuGet.Commands;
+using NuGet.Common;
+using NuGet.Packaging.Signing;
+
+namespace NuGet.MSSigning.Extensions
+{
+    [Command(typeof(NuGetMSSignCommand), "reposign", "RepoSignCommandDescription",
+       MinArgs = 1,
+       MaxArgs = 1,
+       UsageSummaryResourceName = "RepoSignCommandUsageSummary",
+       UsageExampleResourceName = "RepoSignCommandUsageExamples",
+       UsageDescriptionResourceName = "RepoSignCommandUsageDescription")]
+    public class RepoSignCommand : MSSignAbstract
+    {
+        // Default constructor used only for testing, since the Command Default Constructor is protected
```

Comment:
Is this constructor or [this one](https://github.com/NuGet/NuGet.Client/pull/2064/files#diff-cb7c4d75dad42b41c2757e890c95cb71R20) used for testing?